climakitae/explore/vulnerability.py

```diff
@@ -434,48 +490,73 @@ def cava_data(
     display(Markdown(f"**Calculating {names['var_name']}**"))
 
     print(f"--- Selecting Data Points --- \n")
-    data_pts = []
-    for index, loc in input_locations.iterrows():
+    if approach == "time":
 
-        print(f"Selecting data for {loc['lat'], loc['lon']}")
-        lat, lon = loc["lat"], loc["lon"]
+        selections = DataParameters()
+        selections.data_type = "Gridded"
+        selections.downscaling_method = downscaling_method
+        selections.scenario_ssp = (
+            clean_params["ssp_selected"]
+            if historical_data != "Historical Reconstruction"
+            else []
+        )
+        selections.timescale = (
+            "hourly" if downscaling_method == "Dynamical" else "daily"
+        )  # Toggling time-resolution for WRF and LOCA data respectively
+        selections.variable_type = clean_params["variable_type"]
+        selections.variable = clean_params["variable"]
+        selections.resolution = "3 km"
+        selections.units = units
+        selections.scenario_historical = [historical_data]
+        selections.time_slice = (time_start_year, time_end_year)
+
+        if batch_mode:
+
+            separate_files = False
```

Comment:
Just checking - in batch_mode separate_files automatically gets set to False for one single file?

Reply:
Yeah, this is just done with separate files for now since that's the result batch mode comes back as. The next iteration of this can definitely combine the simulations back into their point forms.